h5g_group.go

```diff
@@ -15,6 +15,34 @@ import (
 	"unsafe"
 )
 
+// GType describes the type of an object inside a Group or File.
+type GType byte
+
+const (
+	H5G_GROUP   = GType(C.H5G_GROUP)
+	H5G_DATASET = GType(C.H5G_DATASET)
+	H5G_TYPE    = GType(C.H5G_TYPE)
+	H5G_LINK    = GType(C.H5G_LINK)
+	H5G_UDLINK  = GType(C.H5G_UDLINK)
+)
```

Comment:
Use the same style as for the `IType` constants. I think that with this syntax the consts don't end up associated with the type in the documentation.

Reply:
Yep